src/main/java/com/testsigma/sdk/Email.java

```diff
@@ -15,5 +17,8 @@ public interface Email {
 
   void setContentType(ContentType contentType);
 
+  void setHeaders(String  headers);
```

Comment:
@navyana-srivalli is the method parameter is Map<String, String>?

Reply:
corrected this